frontend/mrs.developer.json

```diff
@@ -4,6 +4,7 @@
     "package": "@plone/volto",
     "url": "git@github.com:plone/volto.git",
     "https": "https://github.com/plone/volto.git",
-    "tag": "18.20.0"
+    "tag": "18.20.0",
+    "filterBlobs": true
```

Comment:
I don't know why this was not the default, I thought I fixed it in cookieplone

Reply:
This is prior to the latest cookieplone